pkg/keto/cmd/create.go

```diff
@@ -227,48 +203,56 @@ func fileExists(f string) bool {
 }
 
 func makeMasterPool(name, clusterName, coreOSVersion, kubeVersion, sshKey, machineType string,
-	diskSize int, networks []string) model.MasterPool {
+	diskSize int, networks, apiNetworks []string) model.MasterPool {
 
 	p := model.MasterPool{}
 	p.Name = name
 	p.ClusterName = clusterName
 	p.CoreOSVersion = coreOSVersion
-	p.KubeVersion = kubeVersion
-	p.SSHKey = sshKey
-	p.Networks = networks
 	p.DiskSize = diskSize
+	p.KubeAPINetworks = apiNetworks
+	p.KubeVersion = kubeVersion
 	p.MachineType = machineType
+	p.Networks = networks
+	p.SSHKey = sshKey
+	// we default to using the master subnets unless specified
+	if len(p.KubeAPINetworks) <= 0 {
+		p.KubeAPINetworks = p.Networks
+	}
+
```

Comment:
Again, not sure we should have an additional flag, this is just confusing. Also defaults like that should not be decided in the CLI, because this is sort of an EC2-specific limitation.